sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,15 +1,18 @@
-# Change Log azure-data-appconfiguration
-## Version 1.0.0-preview.7 (2019-11-26)
+# Release History
+
+## 1.0.0-beta.8 (Unreleased)
+
+## 1.0.0-preview.7
```

Comment:
```suggestion
## 1.0.0-preview.7 (2019-11-26)
```

Reply:
You merged to the repo before merging my suggested change. Perhaps this was missed?

Reply:
Opps, sorry. I'll fix that in another Pr

Reply:
Fixed here https://github.com/Azure/azure-sdk-for-java/pull/6932/files